src/main/java/com/redhat/devtools/lsp4ij/client/features/EditorBehaviorFeature.java

```diff
@@ -181,4 +192,17 @@ public static boolean enableSemanticTokensFileViewProvider(@NotNull PsiFile file
                 ls -> ls.getClientFeatures().getEditorBehaviorFeature().isEnableSemanticTokensFileViewProvider(file)
         );
     }
+
+    /**
+     * Whether or not the document symbols-based breadcrumbs info provider is enabled.
+     *
+     * @param file the PSI file
+     * @return true if the document symbols-based breadcrumbs info provider is enabled; otherwise false
+     */
+    public static boolean enableDocumentSymbolsBreadcrumbsInfoProvider(@NotNull PsiFile file) {
+        return LanguageServiceAccessor.getInstance(file.getProject()).hasAny(
+                file.getVirtualFile(),
+                ls -> ls.getClientFeatures().getEditorBehaviorFeature().isEnableDocumentSymbolsBreadcrumbsInfoProvider(file)
```

Comment:
If we have LSPBreadcrumbfeature, we could write:

` -> ls.getClientFeatures().getBreadcrumbFeature().isEnabled(file)`